### PR TITLE
[PPP-5166] - Use of Vulnerable Component: Components/jackson-databind 2.14.2 (Non OSGi upgrade)

### PR DESCRIPTION
**Pull request by soagarwal1**

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>



---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 48m 5s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl assemblies/prd-ce
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1h 5m 56s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl assemblies/prd-ce
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: assemblies/prd-ce/pom.xml [hunk 1]
```diff
@@ -329,6 +343,20 @@
       <groupId>pentaho</groupId>
       <artifactId>pentaho-metaverse-api</artifactId>
       <scope>compile</scope>
+      <exclusions>
```
Comment (NJtwentyone) [inline assemblies/prd-ce/pom.xml:346]: Per the jira description (https://hv-eng.atlassian.net/browse/PPP-5166)

```
/pentaho-server/pentaho-solutions/system/kettle/plugins/metaverse-plugin/lib/jackson-databind-2.14.2.jar (Can be excluded)
Metaverse has dependencies on jackson-databind, cannot be excluded here, needs to be upgraded.
```
ppp_5166_metaverse_jackson_databind_description



Is the Jira description still valid and/or is jackson-databind still in the lib folder?

**Review by NJtwentyone — CHANGES_REQUESTED** [review]
I have some question https://github.com/pentaho/pentaho-reporting/pull/1698/files#r1765162340
that relates to scope of the this proposed change

File: assemblies/prd-ce/pom.xml (hunk 1 above)
Comment (soagarwal1) [reply assemblies/prd-ce/pom.xml:346]: metaverse is dependent of jackson libraries during compile time and runtime. That is why it is upgraded. https://github.com/pentaho/pentaho-metaverse/pull/698/files#diff-8d04401f1cc51365fe3e32f019cd720135ba920a1a7da7f19e9c9208478701fcR90

Although it isnt present in the metaverse-plugin jar as I can see. I have to look into that. 

But as for prd-ce assembly, jackson-databind is not needed. So excluded.

Comment (NJtwentyone) [reply assemblies/prd-ce/pom.xml:346]: I understand your conclusion for prd-ce.

**Review by NJtwentyone — APPROVED** [review]

**Pull request merged**